adflow/mphys/mphys_adflow.py

```diff
@@ -439,6 +439,23 @@ def set_ap(self, ap):
     def _set_states(self, outputs):
         self.solver.setStates(outputs["adflow_states"])
 
+    def _setup_vectors(self, root_vectors):
+        super()._setup_vectors(root_vectors)
+
+        # Turn off extra printouts
+        self.solver.setOption("printIterations", False)
+        self.solver.adflow.inputiteration.printbcwarnings = False
+
+        # Set the AP
+        self.solver.setAeroProblem(self.ap)
+
+        # Turn printing back on
+        self.solver.setOption("printIterations", True)
+        self.solver.adflow.inputiteration.printbcwarnings = True
```

Comment:
Seems that this was retained after merging (same for above). I think you probably want to restore the previous state as done in PR #307. See [here](https://github.com/mdolab/adflow/pull/307#pullrequestreview-1664677955) for original comment.

Reply:
Good catch.  I'm not sure how I did this, but nonetheless I'll get it fixed.